.github/workflows/ci.yml

```diff
@@ -188,7 +215,7 @@ jobs:
 
       - name: Run tests
         run: |
-          BUILD_JOBS=$((nproc || sysctl -n hw.ncpu) 2> /dev/null)
+          BUILD_JOBS=$( (nproc || sysctl -n hw.ncpu) 2> /dev/null )
```

Comment:
Is this necessary?

Reply:
Is this small whitespace code change necessary? Yes. The script was crashing on Ubuntu 16 without it.  You could confirm, by reverting the change and then run CI.  I believe the reason it wasn't crashing in GitHub Actions before, is because they are providing _newer_ versions of certain packages, a modified environment.

Reply:
I have tried this in my own GHA attempts, and it still doesn't work. I believe, the problem is that `/bin/sh` is `dash` on Ununtu and older Ubuntu versions simply don't support the `$()` syntax, or at least not completely support.

I worked around this by forcing `bash` shell, as described [here](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#defaultsrun) (by setting `defaults.run.shell` to `bash`).